static/js/apps/insights/app.tsx

```diff
@@ -54,7 +54,7 @@ export function App(): JSX.Element {
         setChartData(chartData);
       }
     })();
-  }, [window.location.hash]);
```

Comment:
Why is this removed?

Reply:
`window.location.hash` is outside the scope of the app. I don't think a change to `window.location.hash` will actually result in a re-`fetchData` if it gets changed.

Reply:
I see, ok to remove then.  Thanks for explaining...